LBHackney-IT/lbh-fss-portal-frontend
# Add logout tests

**Description**

Add tests for logout functionality

**Test instructions**

- pull this branch 
- check that logout button works as expected (pressing logout button should redirect the user to the login page)
- run tests using `yarn test` and check tests pass **note** - empty test files have been created for all components, therefore if running all tests you will see multiple failed tests, this merge request just contains tests for the Logout component, therefore please run tests for the Logout component only (i.e. press 'p' to filter tests by filename, and enter 'Logout.test') you should then see 1 test passed (see screenshot below) 

![image](https://user-images.githubusercontent.com/69238887/91051105-8b10ed00-e617-11ea-9ff7-b57b94985854.png)

**Ticket**

https://app.clickup.com/t/9e5u6v
